integration_tests/elemental_03.py

```diff
@@ -30,7 +30,7 @@ def elemental_sqrt32():
     observed: f32[256] = empty(256)
     shape: i32[1] = empty(1, dtype=int)
     eps: f32
-    eps = f32(2e-6)
+    eps = f32(5e-6)
```

Comment:
Why we need to increase the error tolerance here?

Reply:
It has some precision issues with floats in the C backend.